portfolio/src/main/webapp/style.css

```diff
@@ -177,20 +177,37 @@
 #content {
   display: flex;
   align-items: center;
-  justify-content: center;
+  justify-content: space-evenly;
   flex-direction: column;
   margin-left: auto;
   margin-right: auto;
   width: 100vw;
-  height: 100vh;
+  /* height: 100vh; */
   filter: blur(0px);
   -webkit-filter: blur(0px);
 }
 
+#content h1 {
+    margin-top: 30px;
+}
+
 #greeting-container {
   margin-top: 20px;
 }
 
+#comments-container {
+    border-style: solid;
+    border-width: 1px;
+    border-color: #080808;
+    width: 54vw;
+    max-width: 675px;
+    height: 30vh;
+    margin: 10px;
+    padding-top: 20px;
+    margin-top: 30px;
+    overflow: scroll;
```

Comment:
I think the container is already some sort of scroll view now?

Reply:
yes I've added this, the pull request description applies mostly to first commit, i forgot to edit it while committing to this branch.